drivers/dac/ltc2672/ltc2672.c

```diff
@@ -139,39 +139,105 @@ int ltc2672_transaction(struct ltc2672_dev *device, uint32_t comm, bool is_32)
 }
 
 /**
- * @brief set the current for a selected DAC channel
+ * @brief get dac code value from current of a selected DAC channel
  * @param device - ltc2672 descriptor
- * @param current - current to set in mA
- * @param out_ch - channel to set the current
+ * @param dac_current - current in mA
+ * @param out_ch - DAC channel
+ * @return code value
+ */
+uint32_t ltc2672_current_to_code(struct ltc2672_dev *device, float dac_current,
+				 enum ltc2672_dac_ch out_ch)
+{
+	float current;
```

Comment:
you are still using float types. please drop them completely. If you need accurate computations just use a higher resolution for the current values (mA -> uA., etc).